src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/ArtworkAuctionCreateAlertHeader.tsx

```diff
@@ -35,15 +35,18 @@ const ArtworkAuctionCreateAlertHeader: React.FC<ArtworkAuctionCreateAlertHeaderP
   const artistName = artwork.artistNames ? ", " + artwork.artistNames : ""
 
   return (
-    <Box py={12}>
-      <Text variant="lg" textAlign="center">
-        Bidding for <i>{artwork.title?.trim()}</i>
-        {artistName} has ended.
-      </Text>
-      <Box mt={2} mx="auto" width={["100%", 209]}>
+    <GridColumns py={6}>
+      <Column span={12}>
+        <Text variant="lg" textAlign="center">
+          Bidding for <i>{artwork.title?.trim()}</i>
```

Comment:
**suggestion**:  I would move the text to [translations](https://github.com/artsy/force/blob/main/src/System/i18n/locales/en-US/translation.json).

Reply:
Should we also then update the button text then? (we have Create Alert in the translation file) 
I can add this to the next PR since I will have more text there.

Reply:
Yes, I think that we should add all new texts as translations

Reply:
I'm ok to go either way, but we need to be aligned which approach to take. I am asking to add translations because:

1. I was asked to do so many times.
2. I though there was an agreement to do it. 

Regarding backend projects @mzikherman, we had an RFC to adopt internationalization, and resolution was to, in fact, to [adopt it](https://github.com/artsy/README/issues/421#issuecomment-976407182).

I can't find an RFC for Force, maybe we should start one? 🤷